lib/workers/repository/update/pr/changelog/__fixtures__/release-plan.md

```diff
@@ -0,0 +1,146 @@
+# release-plan Changelog
+
+## Release (2025-03-13)
+
+* release-plan 0.16.0 (minor)
+
+#### :rocket: Enhancement
+* `release-plan`
+  * [#155](https://github.com/embroider-build/release-plan/pull/155) add ability to set tag per package
+
+## Release (2025-03-03)
+
+* release-plan 0.15.0 (minor)
```

Comment:
Fixture copied from a real changelog here, but removed committers and user tags
https://github.com/embroider-build/release-plan/blob/a7ffd1f8634bf0fdcb98397a7528000db6b51db7/CHANGELOG.md